package.json

```diff
@@ -32,7 +32,7 @@
     "react": "15.6.1",
     "react-dom": "15.6.1"
   },
-  "author": "",
+  "author": "kingofthestack",
   "homepage": "",
```

Comment:
While we're at it, why not add a homepage? https://kingofthestack.github.io/react-chat-window/